src/operations/booleans/trees/splitLineSegmentByPlane.js

```diff
@@ -0,0 +1,13 @@
+const {vec3} = require('../../../math')
+
+const splitLineSegmentByPlane = (plane, p1, p2) => {
```

Comment:
does this belong here ? or maths ? (unless it is not really reuseable, then here it is fine)

Reply:
I searched the old libraries but could only find this function being used here as part of booleans.